tool/tsh/common/tsh.go

```diff
@@ -819,6 +824,11 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	proxyApp.Arg("app", "The name of the application to start local proxy for").Required().StringVar(&cf.AppName)
 	proxyApp.Flag("port", "Specifies the source port used by by the proxy app listener").Short('p').StringVar(&cf.LocalProxyPort)
 
+	// Local TLS
+	proxyApp.Flag("tls", "Specifies to use TLS Encryption on the local port, uses self-signed if no certificate and key are provided.").BoolVar(&cf.LocalTLS)
+	proxyApp.Flag("local-cert", "Specifies the certificate file for the local TLS Port").Default("").StringVar(&cf.LocalTLSCertificate)
+	proxyApp.Flag("local-key", "Specifies the key file for the local TLS Port").Default("").StringVar(&cf.LocalTLSKey)
```

Comment:
I would rename these to `--cert-file` and `--key-file`.
```suggestion
	proxyApp.Flag("cert-file", "Specifies the certificate file for the local TLS Port").Default("").StringVar(&cf.LocalTLSCertificate)
	proxyApp.Flag("key-file", "Specifies the key file for the local TLS Port").Default("").StringVar(&cf.LocalTLSKey)
```

Reply:
Done